batch/batch/driver/main.py

```diff
@@ -64,6 +64,10 @@
 
 uvloop.install()
 
+google_cloud_profiler_logger = logging.getLogger('googlecloudprofiler')
+google_cloud_profiler_logger.addFilter(adjust_logging_level_error_to_warning)
+import googlecloudprofiler
+
```

Comment:
I'm inclined to be target noisy errors very specifically. The only noise I see from the profiler is failing to collect and upload the profile. That's annoying, but I would otherwise like to know when something else goes wrong. What do you think instead about demoting that particular error to an `info` instead of dropping everything from this dependency?